src/bootstrap/lib.rs

```diff
@@ -32,6 +32,7 @@ use channel::GitInfo;
 use config::{DryRun, Target};
 use filetime::FileTime;
 use once_cell::sync::OnceCell;
+use serde::__private::ToString;
```

Comment:
👀

Reply:
Oops, thanks. That was autoimported by IntelliJ Rust. It's quite confused by bootstrap.

Reply:
can it parse rust_analyzer's config? there's an example config at src/etc/rust_analyzer_settings.json.

Reply:
I don't think it can. It's not that bad, just sometimes it does not understand certain types or performs an "unusual" import :)